src/index.html

```diff
@@ -7,6 +7,68 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class="calendar calendar--month-length--31 calendar--start-day--sun">
+      <li class="calendar__day"></li>
+
+      <li class="calendar__day"></li>
+
+      <li class="calendar__day"></li>
```

Comment:
I prefer not to add gaps between such containers, those tags are the same, I don't see a reason to add a gap between them